drivers/ethernet/eth_enc28j60.c

```diff
@@ -766,7 +766,7 @@ static int eth_enc28j60_init(struct device *dev)
 
 	gpio_pin_interrupt_configure(context->gpio,
 				     config->gpio_pin,
-				     GPIO_INT_EDGE_TO_ACTIVE);
+				     GPIO_INT_EDGE_TO_INACTIVE);
```

Comment:
Do not change it, fix the flag in your [shield overlay](https://github.com/zephyrproject-rtos/zephyr/pull/24988/commits/c17c89be1cc375d581e5eef2a6c4e287258c9d56#diff-23d7bbcd8d979eb604ca1c7d0517f4f6R14)

Reply:
@jfischer-phytec-iot i can add the flags in the shield overlay (line 754-755), but interrupt configure will change it back to high, according to the chip's manual the default is low

Reply:
If the controller pin is connected directly to the MCU then `int-gpios = <&mikrobus_header 7 GPIO_ACTIVE_LOW>;` should work, see also property documentation https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/bindings/ethernet/microchip%2Cenc28j60.yaml#L16

Reply:
i've made the change you requested